index.js

```diff
@@ -52,15 +52,15 @@ controller.hears('invite', 'direct_mention', (bot, message) => {
 /**
  * Private Onboarding
  */
-controller.on('team_join', onboard);
+controller.on('team_join', () => setTimeout(onboard, 5000));
```

Comment:
you are damn right, missed that part, in the actual onboarding script we need to add some sort of timeout to the tests too